### ISSUE TITLE
Hits/visits are not incremented in fullscreen mode not in slideshow

### ISSUE THREAD

**Issue by sppg** [tkuther/piwigo-bootstrap-darkroom#242]
Hi,
  This theme is the perfect one for me, I use it with gdthumb and rv thumbnail scroller.
The only thing I discovered today that when the users navigate in fullscreen mode or when they play the slideshow, the hits/visits counters are not incremented.
Is it a parameter I can adjust myself? or do you need to add it in the code?
Thank you very much

**Comment by sppg** [comment]
Am I the only one who got this issue? is it something I miss configured?

**Comment by dog3n** [comment]
No, I had to register just to tell you that I have the same problem.. When commenting I saw your post was from 2020 ;)

**Comment by sppg** [comment]
> No, I had to register just to tell you that I have the same problem.. When commenting I saw your post was from 2020 ;)

Thank you, at least I don't feel alone about it, maybe now we are 2, solutions may appear ?
